frame/stake-tracker/migrations/src/lib.rs

```diff
@@ -0,0 +1,235 @@
+pub(crate) const LOG_TARGET: &str = "runtime::stake-tracker";
+
+#[macro_export]
+macro_rules! log {
```

Comment:
better put this in the main lib.rs, albeit it makes no difference because of `#[macro_export]`

Reply:
there is a nuance with LOG_TARGET then AFAIR